workers/InstallIlluminaDependencies.R

```diff
@@ -0,0 +1,10 @@
+install.packages("optparse")
+
+install.packages("doParallel", repos="https://rweb.crmda.ku.edu/cran/")
+install.packages("data.table", repos="https://rweb.crmda.ku.edu/cran/")
+
+source("https://bioconductor.org/biocLite.R")
+biocLite("oligo")
+biocLite("limma")
+biocLite("illuminaHumanv2.db")
+biocLite("illuminaHumanv4.db")
```

Comment:
This installation script comes from the upstream project. I'm not R-familiar enough to know about the different decisions made here.

Reply:
@Miserlou: If you don't feel comfortable to deal with R, please add a new issue and I will update the Dockerfile that is used to build the worker_base image.

Reply:
Since the versions aren't specified here, it looks like decisions weren't made about what version to use; it's just using the latest ones. Therefore you should be able to find what the latest versions are and specify them so we know what versions we used to run code if anyone ever needs to know.

Reply:
Moved this all to the main file.